changelog.txt

```diff
@@ -1,5 +1,6 @@
 v.Next
 ----------
+- [MINOR] Enhancing Logging (#1642)
```

Comment:
You're doing more than just enhancing the logging.  You are validating the value stored in the cache looks like JSON.  The Json Syntax exception we receive is telling us it's not JSON.  I'm wondering whether rather than changing how the code will potentially behave and hardcoding rules for what the JSON should look like.  ... Instead you can run your heuristics after the failure.

Reply:
Ok I can update the description. After the failure does make sense, I was putting these conditions to gain more insight as to what specific paring failure there might be to avoid putting PII.